### DISCUSSION TITLE
Is there a way to execute Odin code with turning it to an executable?

### DISCUSSION THREAD

**Discussion by DreepyYunky** [odin-lang/Odin#2346] [Q&A] [answered]
I just wanna know, is there a way to execute code without turning it into an executable

**Comment by z64** [comment, marked as answer]
Currently no, Odin doesn't have something like an interpreted mode that would obviate the need for generating some kind of executable file. While not impossible, that would be a large effort, and there are no comitted plans on this to my knowledge.

The `odin run` command facilitates building & running a program in one step, if that is what you are looking for. It is of course just a "macro" that builds the executable, then runs it, in the current directory.
